sunpy/data/data_manager/tests/conftest.py

```diff
@@ -55,6 +55,16 @@ def manager(tmp_path, downloader, storage, mocker):
     yield manager
 
 
+@pytest.fixture
+def sqlmanager(tmp_path, downloader, sqlstorage, mocker):
```

Comment:
What is the difference between this and the original fixture?

Reply:
Ah, I see, it uses the sqlstorage instead of the inmem one.

Reply:
This one uses an `sqlstorage` instance for the `cache`, instead of `storage` which was an instance of `InMemStorage`. Since this bug pertained to deleting the SQLite db file, I needed a `manager` that uses `sqlstorage`.